### ISSUE TITLE
Error 500 when visit /issues and /pulls (too many SQL variables)

### ISSUE THREAD

**Issue by ghost** [go-gitea/gitea#7750]
- Gitea version (or commit ref): 1.9.0 / 76408d50fb
- Git version: 2.20.1
- Operating system: macOS
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist: 
```
routers/user/home.go:269:Issues() [E] CountIssuesByRepo: too many SQL variables
```
## Description

The above error should be able to explain this issue.

https://github.com/go-gitea/gitea/blob/cd238bc41598e69a78b4d17370c8929911c5f249/models/issue.go#L1357-L1366

https://github.com/go-gitea/gitea/blob/cd238bc41598e69a78b4d17370c8929911c5f249/models/issue.go#L1649-L1658

https://github.com/go-gitea/gitea/blob/cd238bc41598e69a78b4d17370c8929911c5f249/models/issue.go#L1708-L1710

**Comment by lunny** [comment]
This should be closed by #8347

**Comment by guillep2k** [comment]
This should be reopened as per #8427
